ocfweb/stats/templates/stats/printing/pages-printed.html

```diff
@@ -6,6 +6,23 @@
     <div class="ocf-content-block">
         {% stats_navbar %}
         <div id="chart"></div>
+        <div class="row">
+            <div class="col-md-12">
+                <table class="table table-hover table-responsive">
+                    <tr>
+                        <th>Printer</th>
+                        <th>Trees killed over lifetime</tr>
```

Comment:
s/</tr>/</th>/

also I'd give these a more explicit title with an explanation of how we got the numbers, e.g. "Lifetime trees killed per printer assuming 8333 pages per tree" or something similarly descriptive but less verbose

Reply:
Where would be a good place to add the explanation? Adding it within the table heading feels like it would make it too clunky to read.

Reply:
Try putting `<small>` text after/beneath the table.